language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/rename/RenameUtil.java

```diff
@@ -109,7 +109,9 @@ public static Optional<Range> prepareRename(PrepareRenameContext context) {
         // Check if symbol at cursor is empty
         Optional<Symbol> symbolAtCursor = ReferencesUtil.getSymbolAtCursor(context);
         if (symbolAtCursor.isEmpty()
-                || symbolAtCursor.get().kind() == SymbolKind.RESOURCE_METHOD) {
+                || symbolAtCursor.get().kind() == SymbolKind.RESOURCE_METHOD
```

Comment:
Can break int o another if block and extract `symbolAtCursor.get().kind()` out to a variable?

Reply:
This is just a temporary change, we are removing this when we implement the renaming resource path functionality in LS

Reply:
This is just a temporary change, we are removing this when we implement the renaming resource path functionality in LS.